snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -89,9 +89,13 @@ def run_job(self, job: JobExecutorInterface):
         # generic part of a submission string:
         # we use a run_uuid as the job-name, to allow `--name`-based
         # filtering in the job status checks (`sacct --name` and `squeue --name`)
+        if wildcard_str == "":
+            comment_str = f"rule_{job.name}"
+        else:
+            comment_str = f"rule_{job.name}__wildcards_{wildcard_str}"
```

Comment:
Other than the doubled underscore in this line, it looks fine.

Reply:
Done